x-pack/plugins/security_solution/public/timelines/components/graph_overlay/index.tsx

```diff
@@ -178,10 +191,13 @@ const GraphOverlayComponent = ({
       </EuiFlexGroup>
 
       <EuiHorizontalRule margin="none" />
-      <StyledResolver
-        databaseDocumentID={graphEventId}
-        resolverComponentInstanceID={currentTimeline.id}
-      />
+      {graphEventId !== undefined && (
```

Comment:
Resolver cannot work without a `databaseDocumentID` so we hold off rendering it unless we have one.

Reply:
Is `<Suspense>` still not ready yet?

Reply:
I forgot that even existed.